pontoon/contributors/static/js/settings.js

```diff
@@ -1,4 +1,118 @@
 $(function () {
+  $('.field .input')
+    .unbind('keydown.pontoon')
+    .bind('keydown.pontoon', function (e) {
+      if (e.which === 13) {
+        $(this).trigger('blur');
+        return false;
+      }
+    });
+
+  $('.field .input').on('blur', function (e) {
+    e.preventDefault();
+
+    const self = $(this);
+    const value = self.val().trim();
+    const attribute = self.data('attribute');
+    const originalValue = self.data('original-value') || '';
+
+    if (value === originalValue) {
+      return;
+    }
+
+    $.ajax({
+      url:
+        '/user/' +
+        $('#profile input[name="username"]').val() +
+        '/attributes/field/',
+      type: 'POST',
+      data: {
+        csrfmiddlewaretoken: $('body').data('csrf'),
+        attribute: attribute,
+        [attribute]: value || '',
+      },
+      success: function () {
+        self.data('original-value', value);
+        self.attr('data-original-value', value);
+        self.parents('.field').find('.errorlist').empty();
+
+        // contact_email special case
+        if (attribute === 'contact_email') {
+          if (value !== '') {
+            self.parents('.field').find('.help').addClass('hide');
+            self.parents('.field').find('.verify').removeClass('hide');
+          } else {
+            self.parents('.field').find('.help').removeClass('hide');
+            self.parents('.field').find('.verify').addClass('hide');
+          }
+        }
+
+        const message = 'Settings saved.';
+        Pontoon.endLoader(message);
+      },
+      error: function (response) {
+        self.data('original-value', value);
+        self.attr('data-original-value', value);
+
+        // contact_email special case
+        if (attribute === 'contact_email') {
+          self.parents('.field').find('.help').addClass('hide');
+          self.parents('.field').find('.verify').addClass('hide');
+        }
+
+        const errors = response.responseJSON.errors;
+
+        const errorContainer = self.parents('.field').find('.errorlist');
+        errorContainer.empty();
+        for (const error in errors) {
+          const errorMessages = errors[error].join(', ');
+          errorContainer.append($('<p>').text(errorMessages));
+        }
```

Comment:
It's almost never the right choice to use `for...in` iteration.

```suggestion
        const errorContainer = self.parents('.field').find('.errorlist');
        errorContainer.empty();
        for (const error of response.responseJSON.errors) {
          errorContainer.append($('<p>').text(error.join(', ')));
        }
```